dd4t-mvc-support/src/main/java/org/dd4t/mvc/controllers/AbstractBinaryController.java

```diff
@@ -203,12 +203,21 @@ private void fillResponse (final HttpServletRequest request, final HttpServletRe
 
     private static void writeResizedImage (final Binary binary, final String path, final int resizeToWidth, final File binaryFile) throws IOException {
         final File tempBinary = new File(path + ".tmp");
-        saveBinary(binary, tempBinary);
+
 
         try (InputStream content = new FileInputStream(tempBinary)) {
             BufferedImage before = ImageIO.read(content);
```

Comment:
Line 206 : saveBianry(...) method removed, so binary dont get copied to tempBinary. This leads to file not found at line 208 when we try to access.